drivers/console/semihost_console.c

```diff
@@ -6,7 +6,6 @@
 #include <kernel.h>
 #include <device.h>
 #include <init.h>
-#include <kernel_arch_interface.h>
```

Comment:
Yes. `kernel/include/kernel_arch_interface.h` is no longer in the include paths so it would result in built error. Luckily, nothing in that header file is being used, so it is removed.

Reply:
can this be described in commit message?

Reply:
Added a separate commit for the removal.